### PR TITLE
server: Add parameter to start only PosgreSQL

### PR DESCRIPTION
**Pull request by pecigonzalo**
Related to: https://github.com/sourcegraph/sourcegraph/issues/12229

When restoring a deployment DB to a new deployment, we need to start the DB service only to ensure we can restore on a clean database.

### CODE REVIEW

File: cmd/server/shared/postgres.go [hunk 1]
```diff
@@ -153,7 +153,7 @@ func postgresProcfile() (string, error) {
 			}
 		}
 	}
-
+	l("Finished initializing the internal database.")
```
Comment (daxmc99) [inline cmd/server/shared/postgres.go:156]: Not sure if this `l` is a function or a typo... which makes me think that we should rename it if its a function 😂

Comment (pecigonzalo) [reply cmd/server/shared/postgres.go:156]: We have a function called `l` 🤷

Comment (daxmc99) [reply cmd/server/shared/postgres.go:156]: 👍 from me to do a quick rename as part of this PR but otherwise LGTM

Comment (pecigonzalo) [reply cmd/server/shared/postgres.go]: Added

**Pull request merged**